Source/Core/StyleSheetParser.h

```diff
@@ -109,7 +109,9 @@ class StyleSheetParser {
 		const SharedPtr<const PropertySource>& source);
 
 	// Attempts to parse the properties of a @media query
-	bool ParseMediaFeatureMap(PropertyDictionary& properties, const String& rules, bool &match_value);
+	// @param properties Properties to parse into
+	// @param modifier Output modifier
```

Comment:
Here is one attempt to expand this a bit:
```suggestion
    /// Attempts to parse the properties of a @media query.
	/// @param[in] rules The rules to parse.
	/// @param[out] properties Parsed properties representing all values to be matched.
	/// @param[out] modifier Media query modifier.
```